openfisca_us/variables/gov/states/md/tax/income/subtractions/md_child_dependent_care_deduction.py

```diff
@@ -0,0 +1,25 @@
+from openfisca_us.model_api import *
+
+
+class md_child_dependent_care_deduction(Variable):
```

Comment:
Does MD distinguish between subtractions and deductions? We typically organize variables and parameters into a `deductions` folder.